pkg/loader/compose/v1v2.go

```diff
@@ -277,3 +322,80 @@ func checkLabelsPorts(noOfPort int, labels string, svcName string) error {
 	}
 	return nil
 }
+
+func findDuplicateVol(vols kobject.Volumes, volumes []kobject.Volumes) bool {
+	for _, v := range volumes {
+		if vols.MountPath == v.MountPath {
+			return true
+		}
+	}
+	return false
+}
+
+func handleVolume(svcName string, komposeObject kobject.KomposeObject) (volume []kobject.Volumes) {
+
+	if komposeObject.ServiceConfigs[svcName].VolumesFrom != nil {
+		for _, depVol := range komposeObject.ServiceConfigs[svcName].VolumesFrom {
+
+			dVols := handleVolume(depVol, komposeObject)
+			var cVols []kobject.Volumes
+			for _, v := range komposeObject.ServiceConfigs[svcName].Volumes {
+				vol := kobject.Volumes{
+					SvcName:   svcName,
+					MountPath: v,
+				}
+				cVols = append(cVols, vol)
+			}
+			for _, cv := range cVols {
+				i, dv := getVol(cv, dVols)
+				if i == true {
```

Comment:
also in a conditional where you are using boolean values no need to check it with booleans! essentially I mean `s/i == true/i`